Update Authors

certik opened the pull request:
I've added all contributors listed at https://github.com/sympy/sympy-paper/graphs/contributors

They all seem to reasonably satisfy the criteria listed at https://github.com/sympy/sympy-paper/blob/112603990010549031a760dc69a0a385102b49bc/authors/README.md, so that @scopatz can merge it.

@asmeurer, 
@moorepants, 
@Upabjojr, 
@shivamvats, 
@aktech, 
@hargup, 
@scopatz, 
@Sumith1896, 
@leosartaj, 
@fredrik-johansson, 
@thilinarmtb, 
@fabianp,
please update your `email`, `institution` and `institution_address`. Just push into this branch if you can, or send a PR against it.

certik committed:
Update Authors

certik committed:
Move some authos to authors.json.extra

This prevents the following LaTeX error:

! LaTeX Error: Counter too large.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.45

certik commented:
@asmeurer, it looks like the latex template that we are using can't handle more than 9 authors. So I put the rest of the authors into a separate json file (that is currently ignored) for now. Let's still collect all the authors information and merge. We can figure out how to actually make them show in the pdf later.

hargup committed:
Add Harsh Gupta

Signed-off-by: Harsh Gupta <gupta.harsh96@gmail.com>

leosartaj committed:
Add Sartaj Singh

sumith1896 committed:
Add Sumith

fabianp committed:
Update personal data

fabianp commented:
I made PR #103 as I don't have push privileges

thilinarmtb committed:
Update author info for thilinarmtb

aktech committed:
Add AMiT

aktech committed:
Merge pull request #104 from thilinarmtb/authors.json

Update author info for thilinarmtb

aktech committed:
Merge pull request #103 from fabianp/authors.json

Update personal data

asmeurer committed:
Move extra authors json to more reasonable place, and use .json suffix

asmeurer committed:
Make authors.extra.json valid JSON

asmeurer committed:
Add my info (and first author)

shivamvats committed:
Added details of Shivam Vats

scopatz commented on authors/authors.extra.json:
```diff
@@ -0,0 +1,44 @@
+    },
+    {
+        "email": "??",
+        "github_id": "fredrik-johansson",
```
@fredrik-johansson

certik replied:
@fredrik-johansson can you please fill this in? So that we can merge.

scopatz commented on authors/authors.json:
```diff
@@ -1,12 +1,54 @@
     },
     {
+        "email": "??",
+        "github_id": "moorepants",
```
@moorepants

scopatz commented on authors/authors.json:
```diff
@@ -1,12 +1,54 @@
+    },
+    {
+        "email": "??",
+        "github_id": "Upabjojr",
```
@Upabjojr

certik replied:
@Upabjojr can you please fill this in?

certik replied:
I just wrote Francesco an email to fill this in.

scopatz committed:
scopatz edits

scopatz commented:
Just a few more people needed before this is mergable.

moorepants committed:
Added Jason's author details.

certik commented:
@scopatz you broke it. The latex template can't handle more than 9 authors. You need to put yourself into the .extra file for now.

Also, I think you should finish your PR to sympy first. ;)

asmeurer commented:
Someone should make all the address formatting consistent before merging as well.

fredrik-johansson committed:
Update authors.extra.json

asmeurer commented:
By the way, I checked, and the SIAM submission form appears to support as many authors as we have. So we'll just have to work with the journal to make sure they all get listed on the paper itself.

asmeurer committed:
Fix JSON issue

asmeurer committed:
Move an author to authors.extra.json to fix the build

Upabjojr committed:
Update authors.json

isuruf committed:
Fix authors.json

sumith1896 committed:
Change email id to university's, Sumith

certik commented:
Thanks @asmeurer for fixing it. Thanks @Upabjojr for updating your info. Thanks @isuruf for fixing it.

certik commented:
@mattpap, I would love if you can get involved too. You can contact me privately for more info.

ellisonbg committed:
Adding Brian Granger

certik committed:
Move Brian to extra for now

certik committed:
Add Rick Muller into authors

certik committed:
Add Robert Cimrman into authors

certik commented:
@rpmuller, @rc, I just added you to authors. Would you mind checking your information to make sure it is correct?

certik commented:
With this PR, we currently have 18 authors, and there are 19 contributors (i.e. have patches merged into master) to this repository, one of which is @gitter-badger. So we didn't forget anybody so far.

rpmuller commented:
@certik Almost. "name" : "Richard P. Muller", "institution" : "Center for Computing Research, Sandia National Laboratories"

certik committed:
Update Rick's author information

certik commented:
@rpmuller thanks, I fixed it in f00775b5a9ca003495c6145773060e91e9432187.

rc commented:
@certik Thanks, just a few tweaks: 

"institution": "New Technologies - Research Centre, University of West Bohemia"
"institution_address": "Univerzitní 8, 306 14 Plzeň, Czech Republic"

certik committed:
Fix Robert's information

certik commented:
@rc, fixed in cb4b3d54babc475bb7879965bfb025d66d64eba9, thanks!

certik committed:
Move all authors into the same json file

certik committed:
Fix siamart0216.cls to allow more than 9 authors

One just needs to provide enough symbols that are used for the footnotes. For
now I just use numbers 1, 2, 3, ...

This should be addressed by the journal.

certik commented:
I fixed the SIAM template to allow more than 9 authors in ee4644a01bb29ce3bc2456034554385285613fdf. I didn't figure out a fix other than modifying the actual template. But at least it works now.

certik committed:
Use Unicode in authors.json

certik committed:
Create authors.proposed.json

certik committed:
Add results of "git shortlog -ns"

certik committed:
Introduce "np" (number of patches)

This number is taken from the sympy_shortlog.txt file.

certik committed:
Sort the list by number of patches

certik committed:
Filter authors with positive number of patches

certik commented:
I've added a new `np` field into the json, that lists the number of patches to sympy, and then I filter authors with patches > 0, and sort them using this key. It's an arbitrary measure, and we didn't even discuss yet how to order the authors, but ordering them by the number of patches contributed to sympy seems like a sensible first iteration. We can change that later.

@asmeurer, @moorepants, @scopatz would you mind reviewing my latest changes? If all is good, @scopatz can you please merge this? There are a few more authors coming, plus I expect there might be some changes in the `np` field, but we can just send a new PR for that.

moorepants commented:
+1

rc commented:
@certik when I saw `np` I thought numpy - what about using something more descriptive (`num_patches`, `num_commits`, ...)? The other fields are also whole words. Anyway, +1.

aktech commented:
Or Simply name it as `commits` ?

On Tue, May 17, 2016, 1:37 PM Robert Cimrman notifications@github.com
wrote:

> @certik https://github.com/certik when I saw np I thought numpy - what
> about using something more descriptive (num_patches, num_commits, ...)?
> The other fields are also whole words. Otherwise +1.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/sympy/sympy-paper/pull/102#issuecomment-219648024

rc commented:
Yeah, `commits` says it all.

certik commented:
sympy_commits would be even better.

Sent from my mobile phone.
On May 17, 2016 7:56 AM, "Robert Cimrman" notifications@github.com wrote:

> Yeah, commits says it all.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/sympy/sympy-paper/pull/102#issuecomment-219725051

certik committed:
Rename "np" to "sympy_commits"

certik commented:
@rc, @aktech I renamed `np` to `sympy_commits` in 43c534a017c32294ce3cc9965ed175ddac258b98. Thanks for the tip.

rc commented:
Thanks!

certik commented:
Ok, I think @scopatz is too busy, so I am going to take responsibility here and merge it, so that we can move on. I think everybody who is included in this PR (with sympy_patches > 0) clearly passes the authorship criteria, so I think there is no issue with this PR.

certik commented:
Later today I am going to submit PRs with the new authors that are not in yet, unless somebody beats me to it.

asmeurer commented:
Thanks. I actually think you should be in charge of the authors, not @scopatz, since you are spending more time on it.

certik commented:
Ok, I'll be in charge from now on.
